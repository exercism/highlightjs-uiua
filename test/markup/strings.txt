"Hello, world!"
"Hello,\sworld!"
"Hello, \"world\"!"

$"Hello _ !" "World"
$ Hello _ !
$$ Hello _ !
