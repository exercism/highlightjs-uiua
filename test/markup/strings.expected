<span class="hljs-string">&quot;Hello, world!&quot;</span>
<span class="hljs-string">&quot;Hello,\sworld!&quot;</span>
<span class="hljs-string">&quot;Hello, \&quot;world\&quot;!&quot;</span>

<span class="hljs-string">$&quot;Hello <span class="hljs-subst">_</span> !&quot;</span> <span class="hljs-string">&quot;World&quot;</span>
<span class="hljs-string">$ Hello _ !</span>
<span class="hljs-string">$$ Hello <span class="hljs-subst">_</span> !</span>
